src/main/java/randoop/reflection/OperationExtractor.java

```diff
@@ -185,4 +222,195 @@ public void visit(Enum<?> e) {
     EnumConstant op = new EnumConstant(e);
     addOperation(new TypedClassOperation(op, enumType, new TypeTuple(), enumType));
   }
+
+  /**
+   * Indicates whether some omit pattern matches the {@link TypedClassOperation#getRawSignature()}
+   * for the given operation.
+   *
+   * @param operation the constructor or method call to match against the omit patterns of this
+   *     extractor
+   * @return true if the signature matches an omit pattern, and false otherwise.
+   */
+  private boolean omit(TypedClassOperation operation) {
+    if (omitPatterns.isEmpty()) {
```

Comment:
If you cache the result of getRawSignature as suggested elsewhere, you can remove this special-case optimization and reduce code size/clutter a bit.